frenai/chain.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "frenai",
+  "status": "development",
```

Comment:
This is not a valid status. See:
https://github.com/cosmos/chain-registry/blob/099474d5c49ed2dc7ba62a68b96e3f3c96e7284d/chain.schema.json#L36